engine/gui/src/gui.cpp

```diff
@@ -1736,6 +1736,21 @@ Result DeleteDynamicTexture(HScene scene, const dmhash_t texture_hash)
                         lua_settable(L, -3);
                     }
 
+                    if (ia->m_GamepadConnected) 
+                    {
+                        lua_pushliteral(L, "gamepad_name");
+                        if (ia->m_TextCount == 0)
+                        {
+                            lua_pushstring(L, "");
+                        } 
+                        else 
+                        {
+                            lua_pushlstring(L, ia->m_Text, ia->m_TextCount);
+                        }
+
+                        lua_settable(L, -3);
```

Comment:
Prefer the style:
```
lua_pushstring(L, ...);
lua_setfield(L, -2, "gamepad_name");
```

Reply:
Fixed in https://github.com/defold/defold/commit/5a5b367269321c9773c1b9e96d219773ce07b9fe . But I kept `pushlstring`.